### PR TITLE
Create endpoint to reset package mappings of unreleased erratas

### PR DESCRIPTION
**Pull request by Kwaizer**
Resolves: https://github.com/AlmaLinux/build-system/issues/308
**The fontend part of this task needs to be discussed**
- Added an ability to choose an issued_date of erratas

image

Backend: https://github.com/AlmaLinux/albs-web-server/pull/929

### CODE REVIEW

File: src/pages/ErrataFeed.vue [hunk 1]
```diff
@@ -99,6 +99,31 @@
...
+          color="grey-8"
+          @click="resetErratas()"
+          :loading="loadingReset"
+        >
+          Reset
+          <q-tooltip
+            >Reset matched packages of erratas issued after a specified date
+          </q-tooltip>
+        </q-btn>
+      </div>
```
Comment (anfimovdm) [inline src/pages/ErrataFeed.vue:126]: I don't think that place under the page count is the best one

**Review by anfimovdm — CHANGES_REQUESTED** [review]

**Review by anfimovdm — APPROVED** [review]

**Comment by maccelf** [comment]
Could you please do something like this ? 
Screenshot from 2024-08-15 17-06-21
Screenshot from 2024-08-15 17-06-27

File: src/pages/ErrataFeed.vue [hunk 2]
```diff
@@ -65,6 +65,51 @@
         binary-state-sort
         wrap-cells
       >
+        <template v-slot:top-right>
```
Comment (maccelf) [inline src/pages/ErrataFeed.vue:68]: Please show this section only for authorized users

File: src/pages/ErrataFeed.vue [hunk 3]
```diff
@@ -65,6 +65,51 @@
...
+                  Reset matched packages of Advisories after a specified date:
+                </div>
+                <div
+                  class="q-pb-md group row justify-center"
+                  style="gap: 10px; margin: 10px"
+                >
+                  <input
+                    class="date"
+                    type="datetime-local"
+                    style="
```
Comment (maccelf) [inline src/pages/ErrataFeed.vue:97]: Use the style section below to define a new class and set it to the class section for the element

File: src/pages/ErrataFeed.vue [hunk 4]
```diff
@@ -65,6 +65,51 @@
...
+            color="grey-8"
+            @click="showDialogAdvisories = true"
+            :loading="loadingReset"
+          >
+            Reset
+            <q-tooltip
+              >Reset matched packages of erratas issued after a specified date
+            </q-tooltip>
+          </q-btn>
+          <q-dialog v-model="showDialogAdvisories">
```
Comment (maccelf) [inline src/pages/ErrataFeed.vue:82]: Please move this `<q-dialog>` at row 157 (after last `</div>`)

File: src/pages/ErrataFeed.vue [hunk 5]
```diff
@@ -65,6 +65,51 @@
...
         wrap-cells
       >
+        <template v-slot:top-right>
+          <q-btn
+            size="80%"
+            no-caps
+            icon="restart_alt"
+            color="grey-8"
+            @click="showDialogAdvisories = true"
+            :loading="loadingReset"
```
Comment (maccelf) [inline src/pages/ErrataFeed.vue:75]: Move loading from this button to button in dialog

**Review by maccelf — CHANGES_REQUESTED** [review]

File: src/pages/ErrataFeed.vue [hunk 6]
```diff
@@ -65,6 +65,21 @@
...
+          <q-btn
+            size="80%"
+            no-caps
+            icon="restart_alt"
+            color="grey-8"
+            @click="showDialogAdvisories = true"
+          >
+            Reset
+            <q-tooltip
+        <q-tooltip>
```
Comment (maccelf) [inline src/pages/ErrataFeed.vue:78]: Please remove one of the tooltips do the correct indent and make sure that everything works fine

**Review by maccelf — CHANGES_REQUESTED** [review]

**Review by maccelf — APPROVED** [review]

**Pull request merged**